frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.test.tsx

```diff
@@ -0,0 +1,292 @@
+                        createdBy: {
+                            id: 1,
+                            username: 'admin',
+                            imageUrl:
+                                'https://gravatar.com/avatar/21232f297a57a5a743894a0e4a801fc3?size=42&default=retro',
+                        },
+                    },
+                ],
+            },
+        ],
+        approvals: [],
+        rejections: [],
+        comments: [],
+    };
+
+    if (state === 'Scheduled') {
+        result.schedule = {
+            scheduledAt: '2022-12-02T09:19:12.242Z',
+            status: 'pending',
+        };
+    }
+
+    return result;
+};
+const pendingChangeRequest = (changeRequest: IChangeRequest) =>
+    testServerRoute(
+        server,
+        '/api/admin/projects/default/change-requests/pending',
+        [changeRequest],
+    );
+
+const changeRequest = (changeRequest: IChangeRequest) =>
+    testServerRoute(
+        server,
+        '/api/admin/projects/default/change-requests/1',
+        changeRequest,
+        'get',
+    );
+
+const updateChangeRequestState = () =>
+    testServerRoute(
+        server,
+        '/api/admin/projects/default/change-requests/1/state',
+        {},
+        'post',
+    );
+const changeRequestConfig = () =>
+    testServerRoute(
+        server,
+        '/api/admin/projects/default/change-requests/config',
+        [
+            {
+                environment: 'development',
+                type: 'development',
+                changeRequestEnabled: false,
+            },
+            {
+                environment: 'production',
+                type: 'production',
+                changeRequestEnabled: true,
+            },
+        ],
+        'get',
+    );
+
+const setupChangeRequest = (featureName: string, state: ChangeRequestState) => {
+    pendingChangeRequest(mockChangeRequest(featureName, state));
+    changeRequest(mockChangeRequest(featureName, state));
+};
+
+const uiConfig = () => {
+    testServerRoute(server, '/api/admin/ui-config', {
+        versionInfo: {
+            current: { oss: 'version', enterprise: 'version' },
+        },
+        flags: {
+            scheduledConfigurationChanges: true,
+        },
+    });
+};
+
+const user = () => {
+    testServerRoute(server, '/api/admin/user', {
+        user: {
+            isAPI: false,
+            id: 17,
+            name: 'Some User',
+            email: 'user@example.com',
+            imageUrl:
+                'https://gravatar.com/avatar/8aa1132e102345f8c79322340e15340?size=42&default=retro',
+            seenAt: '2022-11-28T14:55:18.982Z',
+            loginAttempts: 0,
+            createdAt: '2022-11-23T13:31:17.061Z',
+        },
+        permissions: [{ permission: ADMIN }],
+        feedback: [],
+        splash: {},
+    });
+};
+const setupOtherServerRoutes = () => {
+    uiConfig();
+    changeRequestConfig();
+    user();
+    updateChangeRequestState();
+};
```

Comment:
Can we move these into setupHttpRoutes? Since that method is calling this method anyway, I don't think this abstraction is necessary.